tools/wptrunner/wptrunner/executors/executormarionette.py

```diff
@@ -1037,8 +1037,13 @@ def __init__(self, logger, browser, server_config, timeout_multiplier=1,
         self.implementation_kwargs = {}
         if reftest_internal:
             self.implementation_kwargs["screenshot"] = reftest_screenshot
-            self.implementation_kwargs["chrome_scope"] = (browser_version is not None and
-                                                          int(browser_version.split(".")[0]) < 82)
+            self.implementation_kwargs["chrome_scope"] = False
+            if browser_version is not None:
+                try:
+                    major_version = int(browser_version.split(".")[0])
+                    self.implementation_kwargs["chrome_scope"] = major_version < 82
```

Comment:
Hm, what exactly is this check for? Maybe you can add a comment?

Reply:
Note that this check already existed; I just made it more robust.

I think the origin is that in older gecko versions you had to switch to chrome scope to use the marionette reftest commands, but I'm not totally sure. _Probably_ we should just delete the check entirely now.